region/crateria/west/Green Pirates Shaft.json

```diff
@@ -1341,6 +1344,7 @@
           "nodes": [2, 3, 4]
         }}
       ],
+      "resetsObstacles": ["A"],
```

Comment:
This strat needs some work, but it's out of scope. I'll see what I can do when I work on resetRoom